@here/harp-mapview/lib/MapView.ts

```diff
@@ -1752,6 +1757,22 @@ export class MapView extends THREE.EventDispatcher {
         return this.m_fog;
     }
 
+    setElevationSource(
+        elevationSource: DataSource,
+        elevationRangeSource: ElevationRangeSource,
```

Comment:
Instead of passing ``elevationRangeSource`` and ``elevationProvider``, why don't we just create them inside the function body? It seems to me like these 2 would be really connected to elevationSource.

Reply:
both are unfortunately in the terrain-datasource gerrit repo, so we have to construct them at the application level. Unless there is a way to pass in the constructor? But I don't know if this is so necessary.

Reply:
another option to simplify it could be to have connect method, similar to the ElevationRangeSource, which would then take the MapView, it would save the application having to pass in the MapView, but then internally I would need to deal with an uninitialized ElevationProvider, so it makes it a bit more code internally.